docs/pages/blog/2021-developer-survey-results.md

```diff
@@ -0,0 +1,406 @@
+  <tr><td>15</td><td>performance - runtime</td><tr>
+  <tr><td>1</td><td>performance - bundle size</td><tr>
+  <tr><td>1</td><td>performance - ?</td><tr>
+  <tr><th>9</th><th>community support</th><tr>
+  <tr><th>5</th><th>icons</th><tr>
+  <tr><th>3</th><th>typescript</th><tr>
+  <tr><th>2</th><th>animations</th><tr>
+</table>
+</details>
+
+### Please rate how much you agree or disagree with the following statements
+
+<img src="/static/blog/2021-survey/4.png" style="width: 796px; margin-top: 16px; margin-bottom: 8px;" alt="I can find most of the components I need (Strongly agree: 45.5%, Agree: 45.9%, Neutral: 6.2%, Disagree: 1.5%, Strongly Disagree 0.8%); I can easily customize the components to match the desired design (Strongly agree: 23.4%, Agree: 46.1%, Neutral: 18.6%, Disagree: 9%, Strongly Disagree 2.9%); I can find the answers to most of my questions in the documentation (Strongly agree: 24.1%, Agree: 50.1%, Neutral: 16.3%, Disagree: 7.6%, Strongly Disagree 1.9%); I find the library's performance to be great (Strongly agree: 26.4%, Agree: 44.3%, Neutral: 22.6%, Disagree: 4.7%, Strongly Disagree 2%); Whenever I needed to get help, I received helpful responses, StackOverflow or GitHub (Strongly agree: 20.9%, Agree: 36.7%, Neutral: 36%, Disagree: 4.6%, Strongly Disagree 1.8%)" />
+
+<p class="blog-description">1534 out of 1589 answered</p>
+
+It's definitely a good thing to have a general sense of agreement to most of the statements. However, there's a considerably big gap between _Strongly agree_ and _Agree_ for all but the first statement, which showcases that we need to continue pushing for even more quality in each of these dimensions (customizability, customization, performance, and support).
+
+### What are your most important criteria for choosing a UI library?
+
+<img src="/static/blog/2021-survey/5.png" style="width: 796px; margin-top: 16px; margin-bottom: 8px;" alt="Ranked list: 1. The design - look and feel; 2. Customizability; 3. Documentation quality; 4. Comprehensiveness; 5. Performance; 6. Popularity; 7. Accessibility; 8. Offered support and help; 9. Bundle size" />
+
+<p class="blog-description">1500 out of 1589 answered - Check <a href="https://www.typeform.com/help/a/ranking-question-360052767651/">Typeform's documentation</a> to understand ranking average calculations.</p>
+
+This year we experimented using Typeform to run the survey, which allowed developers to rank the criteria available - something that wasn't possible with Google Forms. That way, we didn't have to limit the answers to 3 options, allowing to capture a richer perspective. But, even so, the results weren’t too different from last year. Design (look and feel) remains the most important criterion for choosing UI libraries, and customization and documentation quality are still there in the top 3. The notable difference this year seems to be performance climbing its way up, as it has been ranked as the fifth most important aspect.
```

Comment:
```suggestion
This year we experimented with using Typeform to run the survey, which allows respondents to rank their choices. This could've given us a richer perspective than we've been able to capture in the past, but in the end the results weren't too different from last year. 

Design (look and feel) remains the most important criterion for choosing UI libraries, and customization and documentation quality are still there in the top 3. The most notable difference seems to be performance, which climbed to number 5 this year.
```